kpi/views/v2/service_usage.py

```diff
@@ -83,6 +90,9 @@ def list(self, request, *args, **kwargs):
         )
         return Response(data=serializer.data)
 
-    @action(methods=['post'], detail=False)
-    def post(self, request, *args, **kwargs):
-        return self.list(request, *args, **kwargs)
+    def retrieve(self, request, pk=None):
```

Comment:
We'd rather keep DRF signature, i.e. `def retrieve(self, request, pk=None, *args, **kwargs)`. 
(for example, `format` does not work without those parameters)

🤔 Anyway, this endpoint does not seem RESTFul compliant to me. `pk` is not a unique identifier of an `service_usage` model. 

I'd rather see something like:

- `/api/v2/organizations/<pk>/service_usage/`
- `/api/v2/service_usage/`

It may involve more front-end (and back-end) changes but I think it's worth it ;-)